modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/rest-openapi.yaml

```diff
@@ -573,25 +573,487 @@ info:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/CartComment"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/CartComment"
+                    description:
+                        Succesfully update the comment
+                400:
+                    description:
+                        Invalid input
+                401:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Authentication information is missing or invalid
+                404:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        The specified resource was not found
+                500:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Unexpected error
+            summary: Updates an Cart Comment by ID.
+            tags: ["CartComment"]
+    /cart-items/by-externalReferenceCode/{externalReferenceCode}:
+        delete:
+            description:
+                Deletes a Cart Item by external reference code.
+            operationId: deleteCartItemByExternalReferenceCode
             parameters:
                 - in: path
                   explode: false
-                  name: cartCommentId
+                  name: externalReferenceCode
                   required: true
                   schema:
                       description:
-                          Note Id
-                      format: int64
-                      type: integer
+                          Cart Item external Reference Code
+                      example: AB-34098-789-N
+                      minLength: 1
+                      type: string
                   style: simple
             responses:
                 204:
-                    content:
-                        application/json: {}
-                        application/xml: {}
+                    description:
+                        No content
```

Comment:
why the response of this delete is No content compared with the byId where it's application/json: {}